### DISCUSSION TITLE
I am getting this error while running in cigale-No suitable model found for 61200191.

### DISCUSSION THREAD

**Discussion by Mayuri262** [mboquien/cigale#93] [Q&A]
I am getting this error while running in Cigale-No suitable model found for 61200191. It may be that models are older than the universe or that the χ² are very large.
 How to fix this? Do I have to reduce the chi^2 value or something? If so then how? Please guide me I am an undergraduate student and completely new to cigale.

**Comment by mboquien** [comment]
It may be that the redshift is too high so there is no model young enough for the age of the universe. You can try lowering the age of the stellar population. Another possibility is that even the best fit is very poor. This may be that the selected models are not adapted or that there is a photometric problem.
